pegasus/test/test_pdf_merger.rb

```diff
@@ -29,57 +30,89 @@ def setup
     @numbered_collate_file = File.expand_path('../fixtures/numbered_files.collate', __FILE__)
   end
 
+  def assert_local_dependencies
+    [
+      {
+        name: 'gs',
+        url: 'https://www.ghostscript.com/',
+        osx: 'brew install gs',
+        ubuntu: 'sudo apt-get install ghostscript'
+      },
+      {
+        name: 'pdftk',
+        url: 'https://www.pdflabs.com/tools/pdftk-the-pdf-toolkit/',
+        osx: 'brew install https://raw.github.com/quantiverge/homebrew-binary/pdftk/pdftk.rb',
```

Comment:
This command isn't working for OS X as alluded to by the instructions in SETUP.md. I think this is probably fine for now and if we fix the setup instructions we can try to remember to fix it here too.